sdk/core/src/options.rs

```diff
@@ -33,6 +33,7 @@ pub struct ClientOptions {
     /// Telemetry options.
     pub(crate) telemetry: TelemetryOptions,
     /// Transport options.
+    #[allow(dead_code)] // wasm
```

Comment:
I don't love these... I wish we could just make the code conditionally compile based on features.

Reply:
Noted. I'll try to reduce these in future changes.